modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala

```diff
@@ -126,7 +125,7 @@ class packageSpec extends FlatSpec {
   it should "be in Running status after starting" in {
     val p = Process.eval(Task.now(start(seqId)))
     val qs = process(p)(qs1).take(1).runLast.unsafePerformSync.get._2
-    assert(qs.sequences(seqId).status === SequenceState.Running)
+    assert(qs.sequences(seqId).status == SequenceState.Running)
```

Comment:
For some reason, now the compiler couldn't infer that `SequenceState` is an instance of *Equal*, no idea why. I try fiddling but couldn't figure it out. If you know how to fix it, please let me know.